web-console/tsconfig.json

```diff
@@ -50,7 +50,8 @@
   "include": [
     "src",
     "declaration.d.ts",
-    ".next/types/**/*.ts"
+    ".next/types/**/*.ts",
+    "out/types/**/*.ts"
```

Comment:
why do we need to rely on `out/` being there? isn't it only generated on build?

Reply:
It's a recommendation applied automatically by Next 13